cypress/integration/character_exploration.js

```diff
@@ -0,0 +1,97 @@
+describe('Character Management Test Suite', () => {
+    const characterOwner = 'testuser'
+    const characterExplorer = 'testexplorer'
+    const character_name = 'John'
+    const character_race = 'Smith'
+    const character_class = 'Upper'
+    const character_faction = 'Five'
+    const character_level = 5
+    const character_downtime = 30
+    const public_character_status = 'public'
+    let character_exploration_link
+    let character_exploration_id
+
+    before(() => {
+        cy.refreshDatabase()
+    })
+
+    it('Create Public Character', () => {
+        cy.login({name: characterOwner})
+        cy.csrfToken().then((token) =>
+            cy.request('POST', Cypress.Laravel.route('character.store'), {
```

Comment:
so from my understand, are we not using the actual form on the front-end to create a character?

Reply:
No, we are using it to toggle the character's privacy back to "private". This was also just me trying to send requests using Cypress. I thought it'd be good to use both.